elements/pfe-switch/src/pfe-switch.js

```diff
@@ -0,0 +1,178 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeSwitch extends PFElement {
+  static get tag() {
+    return "pfe-switch";
+  }
+
+  get schemaUrl() {
+    return "pfe-switch.json";
+  }
+
+  get templateUrl() {
+    return "pfe-switch.html";
+  }
+
+  get styleUrl() {
+    return "pfe-switch.scss";
+  }
+
+  static get events() {
+    return {
+      change: `${this.tag}:change`
+    };
+  }
+
+  get pfeChecked() {
+    return this.getAttribute("pfe-checked");
+  }
+
+  // Declare the type of this component
+  static get PfeType() {
+    return PFElement.PfeTypes.Content;
+  }
+
+  constructor() {
+    super(PfeSwitch, { type: PfeSwitch.PfeType });
+    this._updateSwitchState = this._updateSwitchState.bind(this);
+    this.settings = {};
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    const $pfeSwitch = this.shadowRoot.querySelector(".pfe-switch");
```

Comment:
The `$name` syntax is a jQuery convention to indicate a jQuery element variable but would not make sense in this context since we do not use jQuery in the project.  The convention we've been using for shadow elements is `this._pfeSwitch = this.shadowRoot.querySelector(".pfe-switch");`.  If you remove the div from the template, you can reference the host simply with `this` and there's no need to get the shadow element at all.